src/test/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByStateITCase.java

```diff
@@ -0,0 +1,283 @@
+      "NA12878	NA12882	0.03663003663003663	10.0	273",
+      "NA12886	NA12890	0.04814814814814815	13.0	270",
+      "NA12882	NA12883	0.026515151515151516	7.0	264",
+      "NA12884	NA12886	0.05776173285198556	16.0	277",
+      "NA12884	NA12887	0.06642066420664207	18.0	271",
+      "NA12879	NA12880	0.04797047970479705	13.0	271",
+      "NA12883	NA12892	0.5729166666666666	165.0	288",
+      "NA12889	NA12892	0.5787671232876712	169.0	292",
+      "NA12880	NA12890	0.05925925925925926	16.0	270",
+      "NA12881	NA12891	0.08118081180811808	22.0	271",
+      "NA12884	NA12888	0.05514705882352941	15.0	272",
+      "NA12880	NA12882	0.040740740740740744	11.0	270",
+      "NA12880	NA12886	0.041044776119402986	11.0	268",
+      "NA12879	NA12883	0.022641509433962263	6.0	265",
+      "NA12887	NA12891	0.06343283582089553	17.0	268",
+      "NA12880	NA12893	0.0599250936329588	16.0	267",
+      "NA12880	NA12885	0.04428044280442804	12.0	271",
+      "NA12879	NA12886	0.05434782608695652	15.0	276",
+      "NA12880	NA12892	0.5655172413793104	164.0	290",
+      "NA12891	NA12892	0.04868913857677903	13.0	267",
+      "NA12883	NA12888	0.5614035087719298	160.0	285",
+      "NA12881	NA12888	0.07352941176470588	20.0	272",
+      "NA12881	NA12886	0.06545454545454546	18.0	275",
+      "NA12889	NA12890	0.05185185185185185	14.0	270",
+      "NA12884	NA12890	0.037037037037037035	10.0	270",
+      "NA12886	NA12889	0.051470588235294115	14.0	272",
+      "NA12885	NA12892	0.05818181818181818	16.0	275",
+      "NA12881	NA12882	0.050724637681159424	14.0	276",
+      "NA12890	NA12891	0.04887218045112782	13.0	266",
+      "NA12879	NA12891	0.058823529411764705	16.0	272",
+      "NA12887	NA12889	0.5778546712802768	167.0	289",
+      "NA12878	NA12891	0.05223880597014925	14.0	268",
+      "NA12878	NA12885	0.03663003663003663	10.0	273",
+      "NA12877	NA12892	0.043795620437956206	12.0	274",
+      "NA12885	NA12891	0.04428044280442804	12.0	271",
+      "NA12879	NA12890	0.04411764705882353	12.0	272",
+      "NA12888	NA12893	0.06319702602230483	17.0	269"
+      };
+
+  static String outputPrefix;
+  static IntegrationTestHelper helper;
+
+  @BeforeClass
+  public static void setUpBeforeClass() throws Exception {
+    helper = new IntegrationTestHelper();
+    outputPrefix = helper.getTestOutputGcsFolder() + "identityByState";
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    for (GcsPath path : helper.gcsUtil.expand(GcsPath.fromUri(outputPrefix + "*"))) {
+      helper.deleteOutput(path.toString());
+    }
+  }
+
+  @Test
+  public void testPaginatedLocal() throws Exception {
+    String[] ARGS = {
+        "--references=" + helper.PLATINUM_GENOMES_BRCA1_REFERENCES,
+        "--variantSetId=" + helper.PLATINUM_GENOMES_DATASET,
+        "--hasNonVariantSegments",
+        "--output=" + outputPrefix,
+        "--useGrpc=false"
+        };
+    testBase(ARGS);
+  }
+
+  @Test
+  public void testPaginatedCloud() throws Exception {
```

Comment:
Please remove this test.  The IdentityByState pipeline only uses gRPC streaming (there is no option to fall back to the paginated request).